e2e_test/src/flight_sql.rs

```diff
@@ -141,4 +141,851 @@ mod test {
 
         assert_batches_eq!(expected, &actual);
     }
+
+    #[tokio::test]
+    async fn test_sql_client_excute_data_type_between() {
+        let mut client = authed_client().await;
+
+        // clean env
+        let flight_info = client
+            .execute("DROP DATABASE IF EXISTS tc_between;".to_string())
+            .await
+            .unwrap();
+        let _actual = fetch_result_and_print(flight_info, &mut client).await;
```

Comment:
There are a lot of such codes in test cases, you can consider extracting a general method

Reply:
```rust
async fn clean_env(client: &mut FlightSqlServiceClient, db_name: &str) {
        let flight_info = client
            .execute(format!("DROP DATABASE IF EXISTS {};", db_name))
            .await
            .unwrap();
        let _actual = fetch_result_and_print(flight_info, client).await;
    }
```